### ISSUE TITLE
Event::fire() not working when testing with Event::fake()

### ISSUE THREAD

**Issue by jonathandey** [octobercms/october#5371]
- OctoberCMS Build: v1.1.*
- PHP Version: 7.4

### Description:
When attemoting to test an `Event::fire` in a unit test I'm recieveing the error `Error: Call to undefined method Illuminate\Support\Testing\Fakes\EventFake::fire()`. I raised this in the [Slack channel](https://octobercms.slack.com/archives/C20UWUV1C/p1606654130121500) and it was pointed out that some work would need to be carried out on the `EventFake` class. [A similar bug was recently fixed](https://github.com/octobercms/library/pull/538) for the `Mail::fake()` method.

### Steps To Reproduce:

1. Create a unit test file withing your plugin.
2. Include the test contents...

```
function testMethod() {
    Event::fake();

    Event::fire(MyCustomEvent::class);

    Event::assertDispatched(MyCustomEvent::class);
}
```

**Comment by mjauvin** [comment]
I confirm the issue, this was broken when Laravel removed the fire method on the event dispatcher class in 5.4

**Comment by bennothommo** [comment]
Fixed by https://github.com/octobercms/library/pull/544.

**Comment by jonathandey** [comment]
@daftspunk Hey! I believe this is broken in v2 - can you confirm?

**Comment by daftspunk** [comment]
> @daftspunk Hey! I believe this is broken in v2 - can you confirm?

Confirmed. Will be fixed in v2.0.26. Thanks for the report. For faster turnaround times, please send bug reports with the platform to hello@octobercms.com
